apps/rush-lib/src/logic/PublishUtilities.ts

```diff
@@ -388,6 +388,10 @@ export class PublishUtilities {
           if (!depProject.shouldPublish || projectsToExclude && projectsToExclude.has(depName)) {
             // No version change.
             return;
+          } else if (prereleaseToken && prereleaseToken.hasValue && prereleaseToken.isPartialPrerelease && depChange.changeType! < ChangeType.hotfix) {
+            // FOr partial prereleases, do not version bump dependecies with the `prereleaseToken`
+            // value unless an actual change (hotfix, patch, minor, major) has occured
+            return
```

Comment:
Tslint should be failing because of the missing semicolon here. If not, could you let us know?

Reply:
It emitted a warning, but not an error. So the build still ran.

Reply:
I fixed the warnings though.

Reply:
Cool